<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reconstruction</class>
 <widget class="QWidget" name="Reconstruction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三维重建</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="openLeftImageButton">
     <property name="text">
      <string>打开左图像</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="rightImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>请点击下方按钮打开右图像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="disparityImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="leftImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>请点击下方按钮打开左图像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="openRightImageButton">
     <property name="text">
      <string>打开右图像</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="getResultButton">
     <property name="text">
      <string>生成视差图</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
